packages/nc-gui/components/dlg/TableCreate.vue

```diff
@@ -78,14 +78,18 @@ const systemColumnsCheckboxInfo = SYSTEM_COLUMNS.map((c, index) => ({
   disabled: index === 0,
 }))
 
+const creating = ref(false)
+
 const _createTable = async () => {
   try {
+    creating.value = true
     await validate()
   } catch (e: any) {
     e.errorFields.map((f: Record<string, any>) => message.error(f.errors.join(',')))
     if (e.errorFields.length) return
   }
   await createTable()
+  creating.value = false
```

Comment:
This line should be put inside `finally` block so that it will set to `false` even there is an error. Otherwise, users  cannot resubmit like below.

![image](https://user-images.githubusercontent.com/35857179/220819121-ed3ac43c-4db3-429d-8224-dac848847a67.png)